### DISCUSSION TITLE
Guest User Management and Control?

### DISCUSSION THREAD

**Discussion by olsch01** [dgreif/ring#1245] [Q&A]
Hi all - I'm trying to find a way to programmatically create a guest user for our Ring security system and also toggle enable lock access for those users as a way to better automate our Airbnb.  Today I manage this by hand within the ring app where I have a Schlage zwave lock integrated.

Glancing through the examples I did not see a way to do that - is something like that possible and if so any examples out there.

Thanks!

**Comment by dgreif** [comment]
I'm sure there are good API endpoints for managing shared users, but I don't think we have them exposed via the `ring-client-api` today. PRs welcome if you manage to track some of them down! Should be fairly easy to discover if you are able to make the changes via ring.com and monitor the network traffic via browser dev tools
